src/block-components/image/index.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! function_exists( 'stackable_load_image_optimizer_polyfill_frontend_script' ) ) {
+	function stackable_load_image_optimizer_polyfill_frontend_script( $block_content, $block ) {
+		// If Easy IO setting is activated for EWWW Image Optimizer, dynamic images becomes blurry.
+		// Load the script to fix the issue.
+		if ( ! is_admin() ) {
+			wp_enqueue_script(
+				'stk-frontend-image-optimizer-polyfill',
+				plugins_url( 'dist/frontend_image_optimizer_polyfill.js', STACKABLE_FILE ),
+				array(),
+				STACKABLE_VERSION,
+				true
+			);
+
+			// Only do this once.
+			remove_action( 'stackable/enqueue_scripts', 'stackable_load_image_optimizer_polyfill_frontend_script', 10 );
+		}
+	}
+
+	if ( ! is_admin() && is_plugin_active( 'ewww-image-optimizer/ewww-image-optimizer.php' )) {
```

Comment:
This would not work if the EWWW folder is renamed. It's would be better if you can use a checker if a function or constant from EWWW is present